ui/src/app/pages/workspace/workspace-card.tsx

```diff
@@ -162,19 +162,18 @@ export class WorkspaceCard extends React.Component<WorkspaceCardProps, Workspace
     };
   }
 
-  async deleteWorkspace() {
-    const {workspace} = this.props;
-    this.setState({
-      confirmDeleting: false,
-      loadingData: true});
-    try {
-      await workspacesApi().deleteWorkspace(workspace.namespace, workspace.id);
-      this.setState({loadingData: false});
-      await this.props.reload();
-    } catch (e) {
-      this.setState({bugReportOpen: true, bugReportError: 'Could not delete workspace', loadingData: false});
```

Comment:
This appears to have been the only usage of these two pieces of state: `bugReportOpen`, `bugReportError`, please remove them and any related code from this class

Reply:
Good catch. Removed.